metricbeat/module/kubernetes/_meta/test/docs/without-ksm/metricbeat-kubernetes.yaml

```diff
@@ -7,66 +7,29 @@ metadata:
   labels:
     k8s-app: metricbeat
 data:
-  metricbeat.yml: |-
+  metricbeat.yml: |- 
```

Comment:
do you mind splitting this file into 1 file per resource. some of the advantages:
- easier to review if you touched a specific resource by looking at the name of the resource modified
- it is easier changing the order of the resource like at https://github.com/elastic/elastic-agent/pull/2104/files#diff-68eb49af89c50af014d3edcec563bc6ef6afbe3c099c5fb2bf177876bce92c72R1. I had a bug in the past that when deploying on Kind and the service account was defined last in the manifest it would take 6m to deploy all the resources compared to a couple of seconds if the resources were in right order (eg. service-account, roles, role-binding...)
- you can even refer to resources defined somewhere else (via a github link) that should be shared. For example you could avoid proving a manifest for a service account resource if already defined somewhere else. this should make it easier to follow the DRY principle